Fix snow so it ends correctly.

noudess opened the pull request:
All packets to end weather should have the type.  The code in command.cpp knows this, the automated weather system did not.  Snow was not ending correctly.

mackal commented:
Can you give me a few steps to reproduce so I can observer? (I know you mentioned them in IRC ...)

noudess commented:
Sure.  So when stow starts in a zone (stonebrunt is where I noticed the bug and fixed it), you get snow visuals and wind (if you're not in a tunnel).  The wind, never stops.  That's because the code sends up to end-rain packet for everything.  That seems to end the visual correctly, but not the audio (wind).  The code in command.cpp for the #weather command works correctly.  I just fixed the automated weather code to send the correct end packet (type and 0 for intensity).  Stonebrunt is working fine now.

mackal commented:
I couldn't get it to snow ... I'll trust you that it fixes it :P

noudess commented:
Takes awhile.  I just plunked someone there with loggin of the packets.

I wonder how they opened Velious..
